save-backend/src/main/kotlin/org/cqfn/save/backend/service/OrganizationService.kt

```diff
@@ -51,9 +44,9 @@ class OrganizationService(private val organizationRepository: OrganizationReposi
      * @param relativePath
      */
     fun saveAvatar(name: String, relativePath: String) {
-        val organization = organizationRepository.findByName(name).apply {
+        val organization = organizationRepository.findByName(name)?.apply {
```

Comment:
Please handle missing organization here (maybe, with an exception). Now you silently ignore missing organization and don't indicate that no avatar has actually been saved.

Reply:
Yes, true. Now throwing exception.